### ISSUE TITLE
Empty invoice emails

### ISSUE THREAD

**Issue by The-Compiler** [python-summit/SPSA#6]
Looks like that Wave thingy sends emails with a plaintext part which is empty, i.e. you just get an empty mail when viewing them in plaintext format.

This caused me to ignore those until now, when I investigated what was going on :wink:

Not sure if there's anything we can do about this, but I guess we should at least let their support know.

**Comment by chiesax** [comment]
Fortunately we are using ti.to for conference tickets then...

Next year I will send the fee invoices directly.

**Comment by dbrgn** [comment]
Non-equivalent HTML and plaintext versions of an email are not standards compliant. But it's not that anyone cares :)
